src/playback/internal/playbackcontroller.cpp

```diff
@@ -1582,6 +1582,7 @@ void PlaybackController::setIsExportingAudio(bool exporting)
 {
     m_isExportingAudio = exporting;
     updateSoloMuteStates();
+    seekRangeSelection();
```

Comment:
Necessary in the case that the user had already started playing through the currently selected range at the point they choose "Export" - need to rewind the playback position to the start of the selection.